rllib/policy/policy.py

```diff
@@ -362,6 +365,11 @@ def __init__(
         self.agent_connectors = None
         self.action_connectors = None
 
+    @ExperimentalAPI
+    @OverrideToImplementCustomLogic
+    def make_rl_module(self) -> "RLModule":
+        """Returns the RL Module"""
```

Comment:
Can you add here that "if this is implemented, then ..." + ""If it's not implemented, then ...".

Reply:
Added.